content/tracing/services/service.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Service
+kind: Documentation
+further_reading:
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+Selecting a service on the services page leads you to the detailed service page:
+
+{{< img src="tracing/services/service/detailed_service_page.png" alt="Detailed service page" responsive="true" popup="true" style="width:90%;">}}
+
+Consult on this page:
+
+* Your [service monitor states](#service-monitor) 
+* Your [out of the box graphs](#out-of-the-box-graphs) 
+* Your [Resources](/tracing/services/resource) 
+
+## Definition
+
+**A Service is the name of a set of processes that do the same job.**  
+For instance, a simple web application may consist of two services: 
+
+* A single `webapp` service and a single `database` service
+
+While a more complex environment may break it out into 6 services: 
+
+* 3 separate services: `webapp`, `admin`, and `query`.
+* 3 separate external service:  `master-db`,  `replica-db`, and `yelp-api`.
+
+You defined those service when instrumenting your application with Datadog.
+An example of setting a custom Service using Python:
+```python
+from ddtrace import tracer
+# add the `wrap` decorator to trace an entire function
+@tracer.wrap(service='my-app')
+```
+
+Service names:
+
+* **must be lowercase, alphanumeric characters**.
+* **cannot have more than 50 characters**.
+* **cannot contain spaces** (spaces are replaced with underscores).
+* must adhere to [metric naming rules](/developers/metrics/).
+
+**Note**: Service must have a type attached, [learn more on Service type]().
+
+## Service monitor
+
+Datadog propose list of monitor depending of your service name/(type?): 
```

Comment:
I don't know about the wording, but we have to be clear these are suggestions not existing by default.

And yes, the suggestions depends on the type of service considered.
Suggested monitors are scoped to the service/env looked at.

Reply:
👌